packages/compiler/src/shadow_css.ts

```diff
@@ -364,7 +364,7 @@ export class ShadowCss {
     unscopedKeyframesSet: ReadonlySet<string>,
   ): CssRule {
     let content = rule.content.replace(
-      /((?:^|\s+|;)(?:-webkit-)?animation(?:\s*):(?:\s*))([^;]+)/g,
+      /((?:^|\s+|;)(?:-webkit-)?animation(?:\s*):(?:\s*))(?:,*)([^;]+)/g,
```

Comment:
I don't know why some of these are non-capturing groups, as there is no grouping going on. Can be simplified to:

```suggestion
      /((?:^|\s+|;)(?:-webkit-)?animation\s*:\s*),*([^;]+)/g,
```

Reply:
I tested your proposed regex locally and tests passed. I updated to use your proposed regex, squashed commits, and force pushed. Files changed looks good. Thank you for reviewing.